README.md

```diff
@@ -1,3 +1,12 @@
+> ⚠️&nbsp;&nbsp;Most Azure resources, including Azure Container Registries can
```

Comment:
```suggestion
> ⚠️&nbsp;&nbsp;Most Azure resources, including Azure Container Registries, can
```

Reply:
Not sure if this is nit: style or hard grammar rule -- defer to you @krancour

Reply:
I agree there should definitely be a comma there. Fixing now.

Reply:
Fixed.